examples/gallery/images/rgb_image.py

```diff
@@ -0,0 +1,39 @@
+"""
+RGB Image
+---------
+The :meth:`pygmt.Figure.grdimage` method can be used to plot Red, Green, Blue
+(RGB) images, or any 3-band false color combination. Here, we'll use
+:meth:`rioxarray.open_rasterio` to read a GeoTIFF file into an
+:class:`xarray.DataArray` format, and plot it on a map.
+
+The example below shows a Worldview 2 satellite image over
+`Lāhainā, Hawaiʻi during the August 2023 wildfires
+<https://en.wikipedia.org/wiki/2023_Hawaii_wildfires#L%C4%81hain%C4%81>`_.
+Data is sourced from a Cloud-Optimized GeoTIFF (COG) file hosted on
+`OpenAerialMap <https://map.openaerialmap.org>`_ under a
+`CC BY-NC 4.0 <https://creativecommons.org/licenses/by-nc/4.0/>`_ license.
+"""
+import pygmt
+import rioxarray
+
+###############################################################################
+# Read 3-band data from GeoTIFF into an xarray.DataArray object
+image = rioxarray.open_rasterio(
+    filename="https://oin-hotosm.s3.us-east-1.amazonaws.com/64d6a49a19cb3a000147a65b/0/64d6a49a19cb3a000147a65c.tif",
```

Comment:
This GeoTiff file is almost 1 GB, which is too big to download.

Reply:
The next line `overview_level=5` gets a reduced resolution version. Original image has shape (y: 115712, x: 99328), but at overview_level=5, the shape is (y: 1808, x: 1552).